docs/src/reference/quadrature.md

```diff
@@ -7,7 +7,9 @@ DocTestSetup = :(using Ferrite)
 
 ```@docs
 QuadratureRule
-AbstractRefShape
```

Comment:
So ref shapes are now gone from docs? :P

Reply:
It was included elsewhere and caused a Documenter warning.